CHANGELOG.md

```diff
@@ -164,6 +164,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (types/errors) [\#10779](https://github.com/cosmos/cosmos-sdk/pull/10779) Move most functionality in `types/errors` to a standalone `errors` go module, except the `RootCodespace` errors and ABCI response helpers. All functions and types that used to live in `types/errors` are now aliased so this is not a breaking change.
 * (gov) [\#10854](https://github.com/cosmos/cosmos-sdk/pull/10854) v1beta2's vote doesn't include the deprecate `option VoteOption` anymore. Instead, it only uses `WeightedVoteOption`.
 * (types) [\#11004](https://github.com/cosmos/cosmos-sdk/pull/11004) Added mutable versions of many of the sdk.Dec types operations.  This improves performance when used by avoiding reallocating a new bigint for each operation.
+* (types) [\#10948](https://github.com/cosmos/cosmos-sdk/issues/10948) Use Tendermint `db-backend` config value instead of compile-time `types.DBBackend` value.
```

Comment:
I added changelog entries for `Improvements` and `Deprecated`, but I didn't add one for `API Breaking`. I couldn't quite tell if it counts as a "Breaking" change. On one hand, the `types.DBBackend` value is no longer really used and doesn't need to be set when compiling. But on the other hand, this PR doesn't do anything that requires a user of the SDK to make changes on their end unless they're using one of the deprecated things.

If someone thinks an entry should be made under `API Breaking` (or something else), let me know.

Reply:
I think we will add `API Breaking` entry in the following release, when we remove `sdk.NewLevelDB`

Creating a task for that: https://github.com/cosmos/cosmos-sdk/issues/11241